### PR TITLE
Adding UI rating on product page including numbers of rating

### PR DESCRIPTION
**Pull request by Tomkndn**
This PR is the fixes for #4 

- Added UI for rating.
- Added numbers on rating.
- Optimised some of the lines.

Screenshot is attached again for new PR:

image

### CODE REVIEW

File: src/pages/Shop/ProductDetail.tsx [hunk 1]
```diff
@@ -12,6 +12,28 @@ interface Data {
...
+  { className: `${commonClasses} mask-half-2` },
+  { className: `${commonClasses} mask-half-1` },
+  { className: `${commonClasses} mask-half-2` },
+  { className: `${commonClasses} mask-half-1` },
+  { className: `${commonClasses} mask-half-2` },
+  { className: `${commonClasses} mask-half-1` },
+  { className: `${commonClasses} mask-half-2` },
+];
+
+const filledStars = 4.5;
```
Comment (pooranjoyb) [inline src/pages/Shop/ProductDetail.tsx:35]: Why is this static? I guess it should be based on the ratings that I provide right?
Please update it.

image

**Review by pooranjoyb — CHANGES_REQUESTED** [review]
Please do the requested changes

File: src/pages/Shop/ProductDetail.tsx (hunk 1 above)
Comment (Tomkndn) [reply src/pages/Shop/ProductDetail.tsx:35]: ohh sorry i thought the rating will be from backend to only display the rating.
I am changing it right now.

Comment (pooranjoyb) [reply src/pages/Shop/ProductDetail.tsx:35]: Na Na, UI pe hi rakho filhaal toh :)

Comment (Tomkndn) [reply src/pages/Shop/ProductDetail.tsx]: review again please

**Comment by Tomkndn** [comment]
Thanks for merging my PR, moreover I want to work more on this project.

**Comment by pooranjoyb** [comment]
> Thanks for merging my PR, moreover I want to work more on this project.

Nice work with the issue @Tomkndn 
Looking forward to more valuable contributions :)) 🚀

**Pull request merged**